Microsoft.Alm.Authentication/Src/Network.cs

```diff
@@ -718,7 +718,7 @@ public async Task SetContent(HttpContent content)
 
                 lock (_syncpoint)
                 {
-                    _mediaType = content.Headers.ContentType.MediaType;
+                    _mediaType = content.Headers.ContentType?.MediaType;
```

Comment:
nit: Not needed since we're inside an `if` block where we've already checked that `content.Headers.ContentType?.MediaType != null`